lib/token.js

```diff
@@ -7,7 +7,8 @@ module.exports.generateToken = function () {
 
 	try {
 		crypto = require('crypto')
-		var secret = require('../config/config').secret
+		//var secret = require('../config/config').secret
+		var secret = process.env.RPSECRET
```

Comment:
How did you get this back? I thought the encrypted env var support is not there.